lib/manageiq/appliance_console/message_configuration_server.rb

```diff
@@ -110,7 +111,7 @@ def create_logs_directory
       def configure_firewall
         say(__method__.to_s.tr("_", " ").titleize)
 
-        AwesomeSpawn.run!("firewall-cmd --add-port=9093/tcp --permanent") # secure
+        AwesomeSpawn.run!("firewall-cmd --add-port=#{server_port}/tcp --permanent")
```

Comment:
Same here with the command injection

```suggestion
        AwesomeSpawn.run!("firewall-cmd", :params => {:add_port => "#{server_port}/tcp", :permanent => nil)
```